test/functional/ui/wildmode_spec.lua

```diff
@@ -64,3 +65,77 @@ describe('command line completion', function()
     ]])
   end)
 end)
+
+describe('External command line completion', function()
+  local screen
+  local items, selected
+
+  before_each(function()
+    clear()
+    screen = Screen.new(25, 5)
+    screen:attach({rgb=true, wildmenu_external=true})
+    screen:set_on_event_handler(function(name, data)
+      if name == "wildmenu" then
+        items = data
+      elseif name == "wildmenu_select" then
+        selected = data[1]
+      elseif name == "wildmenu_hide" then
+        items = nil
+      end
+    end)
```

Comment:
There won't be any of these events if the test doens't attach with ```ext_wildmenu=ture``` just as other externalised items such as ext_tabline etc.